lib/metrics/metric_api.rb

```diff
@@ -19,29 +17,32 @@ def default_timestamp(array)
     # @param availabilities [Array]
     # @example push datapoints of 2 counter metrics
     #    client = Hawkular::Metrics::client::new
-    #    client.push_data(counters: [{:id => "counter1", :data => [{:value => 1}, {:value => 2}]},{:id => "counter2", :data => [{:value => 1}, {:value => 3}]}])
+    #    client.push_data(counters: [{:id => "counter1", :data => [{:value => 1}, {:value => 2}]},
+    #                        {:id => "counter2", :data => [{:value => 1}, {:value => 3}]}])
     # @example push gauge and availability datapoints
-    #    client.push_data(gauges: [{:id => "gauge1", :data => [{:value => 1}, {:value => 2}]}], availabilities: [{:id => "avail1", :data => [{:value => "up"}]}])
+    #    client.push_data(gauges: [{:id => "gauge1", :data => [{:value => 1}, {:value => 2}]}],
+    #                           availabilities: [{:id => "avail1", :data => [{:value => "up"}]}])
     def push_data(gauges: [], counters: [], availabilities: [])
-      gauges.each { |g|
+      gauges.each do |g|
```

Comment:
Minor: since these are short I would have preferred to keep them one-liners (rubocop shouldn't complain about this):

    gauges.each { |g| default_timestamp g[:data] }